api/alert_rules.go

```diff
@@ -0,0 +1,201 @@
+//
+// Author:: Darren Murray (<darren.murray@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+// AlertRulesService is the service that interacts with
+// the AlertRules schema from the Lacework APIv2 Server
+type AlertRulesService struct {
+	client *Client
+}
+
+type alertRuleType string
+
+type alertRuleSeverity int
+
+type AlertRuleSeverities []alertRuleSeverity
+
+// Alert Rule type is an enum in api v2, with only 1 supported value "Event"
+const AlertRuleEventType alertRuleType = "Event"
+
+// String returns the string representation of a Alert Rule integration type
+func (i alertRuleType) String() string {
+	return string(i)
+}
+
+func (sevs AlertRuleSeverities) toInt() []int {
+	var res []int
+	for _, i := range sevs {
+		res = append(res, int(i))
+	}
+	return res
+}
+
+const (
+	AlertRuleSeverityCritical alertRuleSeverity = iota + 1
```

Comment:
Even though I like you used `iota` here, I don't think it is a good idea. What if someone re-orders this enum or, add something in between severities, just like you did below in the schema service.

At least put a comment that says that the enums MUST be 1,2,3,4,5 or, you could just hardcode them. (hardcoding them would be my suggestion)